example/scripts/sample_data_loader.py

```diff
@@ -283,6 +283,35 @@ def load_usage_data_from_csv(file_name):
         conn.commit()
 
 
+def load_table_owner_data_from_csv(file_name):
+    # Load usage data
+    conn = create_connection(DB_FILE)
+    if conn:
+        cur = conn.cursor()
+        cur.execute('drop table if exists test_table_owner_metadata')
+        cur.execute('create table if not exists test_table_owner_metadata '
+                    '(db_name VARCHAR(64) NOT NULL, '
+                    'schema_name VARCHAR(64) NOT NULL, '
+                    'table_name VARCHAR(64) NOT NULL, '
+                    'owners VARCHAR(128) NOT NULL, '
```

Comment:
is varchar(128) sufficient?

Reply:
Can't say for sure, 256 looks like overkill for owners list 😄 
And 128 is the largest field size I've seen at this script

Reply:
you are right :) that is probably around 12 owners, should be good for this sort of use case.